frontend/parle-brand-explorer/style.css

```
@@ -0,0 +1,1098 @@
+    --parle-dark: #fafaf9;
+    --parle-card-bg: #262626;
+    --parle-card-border: #404040;
+    --parle-text: #f5f5f5;
+    --parle-text-muted: #a3a3a3;
+    --parle-shadow: 0 10px 25px -5px rgba(0, 0, 0, 0.5);
+    --parle-glass: rgba(38, 38, 38, 0.9);
+}
+
+/* Global Page Base */
+body {
+    font-family: 'Outfit', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)